src/tools/schema2graph.ts

```diff
@@ -41,9 +41,10 @@ export class SchemaNode {
   // ensure that nested schemas are generated before the references that rely on them.
   refs = new Map<string, SchemaNode>();
 
-  constructor(schema: Schema, name: string) {
+  constructor(schema: Schema, name: string, kotlinName: string) {
```

Comment:
This param looks to be unused. Did you mean to use this for `kotlinAliases` instead?

Reply:
Leftover from a separate attempt to get this working.